pootle/templates/admin/xhr_paid_tasks_report.html

```diff
@@ -18,13 +18,13 @@
       <%
         var amount = row.amount;
```

Comment:
It sets a default, in case the following if conditions are not matched. The same goes for the latter one.